docs/current/278912-guides.mdx

```diff
@@ -0,0 +1,10 @@
+---
+slug: /278912/guides
+---
+
+import GuideIndex from '@site/src/components/molecules/guideIndex.js'
+
+# Guides
+
```

Comment:
Can we add a sentence here? `Learn more about Dagger using the step-by-step walkthroughs and guides in this section.`

Reply:
Yep. Every `.mdx` document can be a combination of markdown and React components. Lines that do not start with `export`, `import`, or a React components, are parsed as markdown.